.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
@Shubhf please remove this fiile and the DS Store file from the PR. They shouldn't be checked in.